docs/README.md

```diff
@@ -16,6 +17,15 @@ way.
 
 ---
 
+## Goal
```

Comment:
@noahweasley : as per your recommendation in chat, I included a section of why anyone would care about the project.